src/fprime_gds/common/handlers.py

```diff
@@ -27,6 +30,42 @@ def data_callback(self, data, sender=None):
         """
 
 
+class DataHandlerPlugin(DataHandler, abc.ABC):
+    """PLugin class allowing for custom data handlers
```

Comment:
(Just mentioning since we talked about it online: ) I actually think it's probably better not to duplicate the `@abstractmethod` – users who read through the code should be able to figure out how to follow the class tree. We should make sure to document in bright bold font that to implement a plugin, one needs to make sure to implement abstract methods of all parents. And I'm sure it errors out if you don't ?

Reply:
Yes, I have a follow-up PR to document this as part of ROSES.  I will evaluate then if this can be fixed by documentation, moving/cloning the method, or a bit of both.